styles/globals.scss

```diff
@@ -1,11 +1,13 @@
-@import './abstracts/variables';
-@import './base/base';
+@import "./abstracts/variables";
+@import "./base/base";
+@import "./base/typography";
+@import "./components/title";
+@import "./components/subtitle";
 
 body {
   background-color: var(--coral);
   margin: 0;
   font-family: var(--poppins);
-  font-size: 1.5em;
 }
```

Comment:
global.scss would be just the entry file with the imports, if you want some base styles for certain tags like the body, header, h1, etc you can put them in the base.scss file. Also keep in mind consistency, I see there is aheader h1 {...} in this file. IS better if you have in base styles for tags on a general level and if you need to style the h1 or h2 inside the header, better do it in the scss file of the header component

Reply:
Ty, then I'm going to clean the global.scss